CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 2.3.0
```

Comment:
vielleicht kannst du auch gleich die version in der package.json anheben dann könnte ich direkt releasen